component/elements.jsx

```diff
@@ -57,11 +58,14 @@ export const Title = ({ children, ...props }) => (
 export const Footer = ({ children }) => <footer>{children}</footer>
 
 const iconStyle = { style: { marignLeft: '2em' } }
-export const Link = colorize(({ icon, children, ...props }) => (
-  <a {...props}>
-    {Icons[icon]
+export const Link = colorize(({ icon, children, ...props }) => {
+  const tag = /^(http|\/api\/)/.test(props.href) ? 'a' : RouterLink
```

Comment:
Here I look if the link should be internal or external, meaning, we should navigate in JS or let the browser do it.

I use the correct `tag` or component depending of the action